Bizyhood/Seo.php

```diff
@@ -0,0 +1,276 @@
+<?php
+
+// Make sure we don't expose any info if called directly
+if ( !function_exists( 'add_action' ) ) {
+	echo 'Hi there!  I\'m just a plugin, not much I can do when called directly.';
+	exit;
+}
+
+/**
+ * This class contains all Bizyhood Shortcodes.
+ */
+ 
+class Bizyhood_Seo
```

Comment:
Isn;nt "Sitemap" as a class name too broad? I think we have a prefix on all classes, just to be sure.